src/components/Initiatives/InitiativeDetailsPage/InitiativeDetailsPage.tsx

```diff
@@ -194,18 +233,49 @@ export const InitiativeDetailsPage = () => {
             variant="outlined"
             aria-label="Filter"
           >
-            filter
+            Filter
           </Button>
         )}
       </Box>
       <Box my={1}>{renderTab(activeTab)}</Box>
       <InitiativeFilterDialog
-        isOpen={isFilterDialogOpen}
-        actionItems={actionItems}
-        initiative={initiative}
+        filtersDefinition={filterDefinitions}
         handleClose={handleFilterDialogClose}
+        isOpen={isFilterDialogOpen}
         setFilter={(filter: InitiativeFilter) => setFilter(() => filter)}
       />
     </Content>
   );
 };
+
+const InitiativeDetailsPageWrapper: React.FC = () => {
+  const { id: initiativeId } = useRouteRefParams(initiativeRouteRef);
+  const cortexApi = useApi(cortexApiRef);
+
+  const { value, loading, error } = useAsync(async () => {
+    return await Promise.all([
+      cortexApi.getInitiative(+initiativeId),
+      cortexApi.getInitiativeActionItems(+initiativeId),
```

Comment:
numeric coercion hax

Reply:
Isn't string vs number fun?! (This is existing code though)

Reply:
I saw, was just calling it out since it brought back memories of "clever code" that ends up confusing people.